### ISSUE TITLE
Training details possibly missing in Paper?

### ISSUE THREAD

**Issue by L-Reichardt** [cardwing/Codes-for-PVKD#26]
What training parameters did you use? This paper has the following sentence, but no further details.
"And with some engineering tricks like finetuning ... our Cylinder3D ... can obtain 71.2 mIoU". Does this also included training hyperparameters?

The original C3D repo. (which is duplicated in this repository), suffers from strong overfitting around epoch ~15 and multiple people have struggled to reproduce the results of C3D [ref. issue](https://github.com/xinge008/Cylinder3D/issues/107).
Besides augmentation, Adam, BS=2, and LR there are no other parameters. This makes me think, that possibly other parameters were used.

- Missing augmentations?
- Was a learning rate scheduler used, or training with constant learning rate
- Weight decay or dropout?

**Comment by cardwing** [comment]
You can try some data augmentations such as instance augmentation (Panoptic-PolarNet: Proposal-free LiDAR Point Cloud Panoptic Segmentation
) and lasermix (LaserMix for Semi-Supervised LiDAR Semantic Segmentation
) which can improve the performance. Deepening or widening the network can also bring performance gains. Finetuning the model on both train and val sets can boost the performance. Model ensemble can also help.

**Comment by L-Reichardt** [comment]
@cardwing Thank you for your answer, I suspected as much. I've been relying too much on the config of the original C3D repo.
